### PR TITLE
Add non-normative definition of a valid MIDI message

### LINKED ISSUE #164: Define a "valid Midi message"
In [`send`](http://webaudio.github.io/web-midi-api/#widl-MIDIOutput-send-void-sequence-octet--data-DOMHighResTimeStamp-timestamp):

>  If data is not a valid sequence or does not contain a valid MIDI message, throw a TypeError 
> exception. 

There should be some definition or at least some pointer to a normative reference that defines what a "valid sequence" or a "valid MIDI message" is. It seems quite important for compatibility that all UAs accept exactly the same messages or byte sequences.

For example [this document](https://www.midi.org/specifications/item/table-1-summary-of-midi-message) warns exactly about this issue and points to the "official MIDI Specifications".

### PR DESCRIPTION
**Pull request by mjwilson-google**
This is to fix #164.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -169,6 +169,33 @@ <h2>
         <dfn>MIDI message</dfn>, <dfn>System Real Time</dfn> and
         <dfn>System Exclusive</dfn> are defined in [[MIDI]].
       </p>
+      <div class="note">
+        The definition of a <dfn>valid MIDI message</dfn> should be
```
Comment (cwilso) [inline index.html]: I think this should be "[the definition of a valid message] is defined in" (not "should be inferred by").  It's defined in the spec.

**Review by cwilso — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (mjwilson-google) [reply index.html]: The MIDI 1.0 spec doesn't explicitly define a valid MIDI message, although it does refer to a "valid Status byte".  Maybe it's not necessary to have an explicit definition?  How does the wording look now?

Comment (cwilso) [reply index.html]: LGTM.  It's true they don't use the term "valid MIDI message", but the structure of a message is clearly defined starting in page 4.  New wording is good.

BTW, I noted the MIDI specref pointer is out of date - the MMA moved the spec on their site and didn't up date it - and I filed a pull request in specref to update it.

Comment (mjwilson-google) [reply index.html]: >the structure of a message is clearly defined starting in page 4.

Good point!

> BTW, I noted the MIDI specref pointer is out of date - the MMA moved the spec on their site and didn't up date it - and I filed a pull request in specref to update it.

Thank you!  We have issue 266 tracking this too, I will post an update there.

**Pull request merged**